xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -146,21 +147,27 @@
         <ReturnType>System.IAsyncResult</ReturnType>
       </ReturnValue>
       <Parameters>
+        <!-- Question for reviewers: GZipStream and DeflateStream have additional attributes in these Parameters: Index, FrameworkAlternate. Where those autogenerated or do I have to investigate them and add them manually myself? If it's the latter, where did the FrameworkAlternate info come from? I suspect it has something to do with the extra params 'cback' and 'state' which this function does not have. -->
```

Comment:
I'll take a look but you shouldn't mess with those. They come from the mdoc reflection tool.

Reply:
Gotcha. Thanks!

Reply:
Question comment removed.